service/node_test.go

```diff
@@ -2334,3 +2341,23 @@ func TestGetVolumeStats(t *testing.T) {
 		t.Errorf("Expected: error, but got: %v", err)
 	}
 }
+
+func TestReachableEndPoint(t *testing.T) {
+	type args struct {
+		endpoint string
+	}
+	tests := []struct {
+		name string
+		args args
+		want bool
+	}{
+		{"Unreachable IP", args{endpoint: "10.255.1.2:100"}, false},
```

Comment:
Wondering if this may ever get used in the GitHub runners in the future.

Reply:
Not sure, but it may possibly be.